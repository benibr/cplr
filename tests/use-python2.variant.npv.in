use-python2.base.in