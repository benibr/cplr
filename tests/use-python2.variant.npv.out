Package python3: -I/usr/include/python3.7m -I/usr/include/x86_64-linux-gnu/python3.7m -lpython3.7m
Package python3 include dir: /usr/include/python3.7m
Package python3 include dir: /usr/include/x86_64-linux-gnu/python3.7m
Package python3 library: python3.7m
Compilation succeeded.
